scrapping/scrapping/items.py

```diff
@@ -0,0 +1,12 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class ScrappingItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
```

Comment:
The `ScrappingItem` class currently does not define any fields. You should define fields corresponding to the data you plan to scrape, such as `title`, `author`, `price`, etc., using `scrapy.Field()`.